model/src/main/java/io/spine/validation/DistinctPolicy.java

```diff
@@ -41,7 +41,7 @@
 
 import static io.spine.protobuf.AnyPacker.unpack;
 import static io.spine.protodata.Ast.isRepeated;
-import static io.spine.protodata.Ast.qualifiedName;
+import static io.spine.protodata.Ast.getQualifiedName;
```

Comment:
This also looks weird.

I realize it's now too late to comment on this, but I definitely liked the version without the getter better.

Reply:
`qualifiedName` used to be a method. Now it's a property, which looks more native in Kotlin code. Since we're in Java in this class, we use the automatic getter for the property.